bigtest.json

```
@@ -0,0 +1,17 @@
+{
+  "port": 24000,
+  "launch": [
+    "chrome.headless"
+  ],
+  "app": {
+    "command": "npm start",
+    "env": {
+      "PORT": 24001
```

```suggestion
      "PORT": 2800
```

The init script chooses a port that conflicts with the proxy port by default. This was fixed in https://github.com/thefrontside/bigtest/pull/667 but has yet to be released